### ISSUE TITLE
[Design] Confirmation email [MVP? DUPLICATE of 327?]

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/reserve-rec-public#68]
#### Description:

Design a confirmation email(s) that can be sent out to park visitors

#### Design considerations:

- Be able to change information at a per park, per offering level. Lots of things come up that may apply to the backcountry offering for example at Golden Ears, but not Frontcountry. 
- There should be a generic template at the beginning, like what Camis has but as PO’s requests different updates we should be able to update them easily and quickly. 
- We don’t want the confirmation letter to be too long with too much information. Our team should be able to review each request for changes. 
- Perfect world: advisories would be included automatically in the confirmation letter.

#### Other considerations:

- Ties into admin

#### Checklist:

- [ ] Draft design
- [ ] Review with Rizzies 
- [ ] Review with content designer
- [ ] Customized messages based on park
- [ ] Make sure to use concise, plain language
- [ ] ...

#### Dependencies

- Blocked by https://github.com/bcgov/reserve-rec-public/issues/168
- Blocking https://github.com/bcgov/reserve-rec-public/issues/56

#### Relevant documentation as reference
- [Research findings and notes](https://app.mural.co/t/bcparks2575/m/bcparks2575/1751671788851/fc5523bcdb36bbd7186579bf15aa0d498ebad1e3) (Mural Board)
- [Ideas for future confirmation emails](https://bcgov.sharepoint.com/:u:/r/teams/08548/Shared%20Documents/General/Project%20Documentation/4.%20Reservations/Confirmation%20and%20reminder%20emails/Confirmation%20Emails%20-%20how%20it%20is%20now%20vs%20ideal%20situation%20in%20the%20future.msg?csf=1&web=1&e=LsUcZv)
- [Examples of confirmation and reminder emails](https://bcgov.sharepoint.com/:f:/r/teams/08548/Shared%20Documents/General/Project%20Documentation/4.%20Reservations/Confirmation%20and%20reminder%20emails?csf=1&web=1&e=XhqzPZ)

#### Notes:
- 
-

#### Definition of Ready: Design ticket
- [ ] Acceptance criteria are included
- [ ] Relevant documentation to context is linked
- [ ] Definition of Done: Design ticket

#### Designs are peer reviewed by another designer
- [ ] Designs are reviewed by developers
- [ ] Design is reiterated based on feedback, as needed

#### PO Review pipeline:
- [ ] Acceptance criteria are checked by Product Owner
- [ ] Approved by Product Owner

**Comment by Dianadec** [comment]
First iteration already completed. We also have the more finalized version. Question - do we need a second iteration design. Keeping in backlog for now but this ticket may not be needed.

**Comment by LindsayMacfarlane** [comment]
Have repurposed this ticket for design of confirmation email.
